Client/Frontend/Browser/BrowserViewController/ProductNotifications/BrowserViewController+ProductNotification.swift

```diff
@@ -130,6 +131,24 @@ extension BrowserViewController {
                 }
             }
         }
+        
+        // Step 5: Domain Specific Data Saved
+        // Data Saved Pop-Over only exist in JP locale
+        if Locale.current.regionCode == "JP" {
+            if !Preferences.ProductNotificationBenchmarks.domainSpecificDataSavedShown.value {
```

Comment:
This pref name is confusing, it reads like we show the domain-data only once.
I would rename it to something like `showingSpecificDataSavedEnabled`

Reply:
Renamed.!